apps/ledger-live-mobile/src/components/HeaderRightClose.js

```diff
@@ -34,22 +34,30 @@ export default function HeaderRightClose({
   const [onModalHide, setOnModalHide] = useState();
 
   function close(): void {
-    if (onClose) {
-      onClose();
-    }
-
     if (skipNavigation) {
+      // onClose should always be call at the end of the close method,
```

Comment:
Nitpicking but I think it's : `onClose should always be called at the end of the close method`